CHANGELOG.md

```diff
@@ -0,0 +1,79 @@
+# Changelog
+
+<!-- 
+Generate the changelog the using lerna-changelog tool, i.e
+
+./node_modules/lerna-changelog/bin/cli.js --ignoreCommitters (--from core-0.3.0)
```

Comment:
Generate the changelog the using lerna-changelog tool, i.e ./node_modules/lerna-changelog/bin/cli.js --ignoreCommitters (--from core-0.3.0) -> Generate the changelog using the lerna-changelog tool. For example, ./node_modules/lerna-changelog/bin/cli.js --ignoreCommitters (--from core-0.3.0).